### PR TITLE
Update tuya.markdown

### PR DESCRIPTION
**Pull request by paperclipmaximizer**
Users describe trouble authenticating after proper project configuration and app credentials (See https://www.reddit.com/r/homeassistant/comments/q3hnlz/tuya_login_error_1106_permission_deny/). A tested work-around involves adding a custom user to the project in the cloud interface. 

My suggestion is to add a reference to this "solution" or "work-around" to the documentation. I have made the addition to the trouble logging in section of the document. 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/tuya.markdown [hunk 1]
```diff
@@ -132,9 +132,11 @@ If no devices show up in Home Assistant:
...
+    - Incorrect username or password: Enter the correct account and password of the Tuya Smart or Smart Life app in the **Account** and **Password** fields (social login, which the Tuya Smart app allows, may not work, and thus should be avoided for use with the Home Assistant integration). Note that the app account depends on which app (Tuya Smart or Smart Life) you used to link devices on the [Tuya IoT Platform](https://iot.tuya.com/cloud/). 
 
     - Incorrect country. You must select the region of your account of the Tuya Smart app or Smart Life app.
+    
+    - Note: Some users still experience this error after adding correct app account credentials in a correctly configured project. A workaround involves adding a custom user under Cloud > Development > Users.
```
Comment (c0ffeeca7) [inline source/_integrations/tuya.markdown]:
```suggestion
    - Note: Some users still experience the **Permission denied** error after adding correct app account credentials in a correctly configured project. A workaround involves adding a custom user under **Cloud** > **Development** > **Users**. 
```

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]

File: source/_integrations/tuya.markdown [hunk 2]
```diff
@@ -132,9 +132,11 @@ If no devices show up in Home Assistant:
...
+    - Incorrect username or password: Enter the correct account and password of the Tuya Smart or Smart Life app in the **Account** and **Password** fields (social login, which the Tuya Smart app allows, may not work, and thus should be avoided for use with the Home Assistant integration). Note that the app account depends on which app (Tuya Smart or Smart Life) you used to link devices on the [Tuya IoT Platform](https://iot.tuya.com/cloud/). 
 
     - Incorrect country. You must select the region of your account of the Tuya Smart app or Smart Life app.
+    
+    - Note: Some users still experience the **Permission denied** error after adding correct app account credentials in a correctly configured project. A workaround involves adding a custom user under **Cloud** > **Development** > **Users**.
```
Comment (frenck) [inline source/_integrations/tuya.markdown]:
```suggestion
    - Some users still experience the **Permission denied** error after adding the correct app account credentials in a correctly configured project. A workaround involves adding a custom user under **Cloud** > **Development** > **Users**. 
```

File: source/_integrations/tuya.markdown [hunk 3]
```diff
@@ -132,9 +132,11 @@ If no devices show up in Home Assistant:
...
 
-    - Incorrect username or password: Enter the correct account and password of the Tuya Smart or Smart Life app in the **Account** and **Password** fields (social login, which the Tuya Smart app allows, may not work, and thus should be avoided for use with the Home Assistant integration). Note that the app account depends on which app (Tuya Smart or Smart Life) you used to link devices on the [Tuya IoT Platform](https://iot.tuya.com/cloud/).
+    - Incorrect username or password: Enter the correct account and password of the Tuya Smart or Smart Life app in the **Account** and **Password** fields (social login, which the Tuya Smart app allows, may not work, and thus should be avoided for use with the Home Assistant integration). Note that the app account depends on which app (Tuya Smart or Smart Life) you used to link devices on the [Tuya IoT Platform](https://iot.tuya.com/cloud/).
```
Comment (frenck) [inline source/_integrations/tuya.markdown]: Unrelated/unneeded change

```suggestion
    - Incorrect username or password: Enter the correct account and password of the Tuya Smart or Smart Life app in the **Account** and **Password** fields (social login, which the Tuya Smart app allows, may not work, and thus should be avoided for use with the Home Assistant integration). Note that the app account depends on which app (Tuya Smart or Smart Life) you used to link devices on the [Tuya IoT Platform](https://iot.tuya.com/cloud/).
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @paperclipmaximizer! :+1:

**Pull request merged**